### PR TITLE
Yury_Vislobodsky Module 4: Information handling

### PR DESCRIPTION
**Pull request by YurasicV**

### CODE REVIEW

File: Yuri_Vislobodsky/Module 4 -  Information handling/javase04_mvn/pom.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>javase04</groupId>
+    <artifactId>javase04_mvn</artifactId>
```
Comment (technicaltaskjava) [inline Yuri_Vislobodsky/Module 4 -  Information handling/javase04_mvn/pom.xml]: please do not use _mvn as it's do not give any additional info just confusion.

Comment (YurasicV) [inline Yuri_Vislobodsky/Module 4 -  Information handling/javase04_mvn/pom.xml]: ОK, I understood

File: Yuri_Vislobodsky/Module 4 -  Information handling/javase04_mvn/src/main/java/javase04/t01/textbook/TextBook.java [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+/**
+ * TextBook handling class
+ * Created by Yury Vislobodsky on 18.03.2016.
+ */
+public class TextBook extends FileHandler {
+    /**
+     * swap first and last word for every sentence.
+     */
+    public void methodA() {
+        Matcher matcher = Pattern.compile("[А-ЯA-Z](\\d\\.|[^\\.!\\?])+[\\.!\\?]").matcher(content.toString());
```
Comment (technicaltaskjava) [inline Yuri_Vislobodsky/Module 4 -  Information handling/javase04_mvn/src/main/java/javase04/t01/textbook/TextBook.java]: Move pattern out of method.

File: Yuri_Vislobodsky/Module 4 -  Information handling/javase04_mvn/src/main/java/javase04/t01/textbook/TextBook.java [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+    }
+
+    /**
+     * delete all the words of given length, starting with consonant letter
+     */
+    public void methodC(int wordLength) {
+        wordLength = (wordLength > 1) ? wordLength : 1;
+        int offset = 0;
+        Matcher matcher = Pattern
+                .compile("\\s+([Б-ДЖЗК-НП-ТФ-Щб-джзк-нп-тф-щB-DF-HJ-NQ-TV-Zb-df-hj-nq-tv-z][А-Яа-я\\w]{"
```
Comment (technicaltaskjava) [inline Yuri_Vislobodsky/Module 4 -  Information handling/javase04_mvn/src/main/java/javase04/t01/textbook/TextBook.java]: Move pattern out of method to the constant.

**Pull request merged**